src/__tests__/snapshot-tests/__snapshots__/documenter.test.ts.snap

```diff
@@ -88749,7 +88726,7 @@ To find a specific row use the \`findRow(n)\` function as chaining \`findRows().
           },
         },
       ],
-      "name": "AttributeEditorWrapper",
+      "name": "ChartPopoverSeriesWrapper",
     },
```

Comment:
Git diff cannot detect new classes inserted in the middle of the list, and it renders them as "modified". See below, the `AttributeEditorWrapper` still exists